manual/src/cmds/tsan.etex

```diff
@@ -0,0 +1,227 @@
+\chapter{Runtime detection of data races with ThreadSanitizer}
+\label{c:tsan}\cutname{tsan.html}
+%HEVEA\cutname{tsan.html}
+
+\section{s:tsan-overview}{Overview and usage}
+
+OCaml, since version 5.0, allows shared-memory parallelism and thus mutation of
+shared data. This creates the possibility of data races, i.e., unordered
+accesses to the same data (one of them being a write). Data races can be
```

Comment:
"unordered access to the same memory location with at least one of them being a write".